stories/atoms/components/typography/Typography.stories.mdx

```diff
@@ -246,6 +246,25 @@ Internally, this parameter will actually select a variant of the selected font.
   </Story>
 </Canvas>
 
+## Text decoration
+
+It is possible to add a text decoration with the optional property `textDecoration`.  
+This property can take the following value :
+
+- **underline** : underlines the text in the same color as the text itself.
+
+There will be more text decoration as the library develops.
+
+<Canvas>
+  <Story name="Text decoration">
+    <div>
+      <Typography as="div" fontFamily="brand" fontSize="medium" textDecoration="underline">
```

Comment:
Try to avoid passing values that are the default for code readability. Also I propose centering the text to make it look nicer on small screens.

```suggestion
    <div style={{ textAlign: 'center' }}>
      <Typography textDecoration="underline">
```